message.go

```diff
@@ -14,5 +21,15 @@ type Message struct {
 }
 
 func (m Message) String() string {
-	return fmt.Sprintf("%s\t%s\t%s\t%s", m.Timestamp.Format(time.RFC3339), m.Level, m.Labels, m.Message)
+	var Color string
+	if m.Level == "debug" {
+		Color = DebugColor
+	} else if m.Level == "warning" {
+		Color = WarningColor
+	} else if m.Level == "error" {
+		Color = ErrorColor
+	} else {
+		Color = ResetColor
+	}
```

Comment:
If you go forward with this, consider using a map instead of this if-then "waterfall".

Reply:
Or, perhaps, as the linter suggests, a `switch` statement.  😆